projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_job_runner.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import json
+import logging
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from vdk.plugin.meta_jobs.api.meta_job import IMetaJobInput
+from vdk.plugin.meta_jobs.meta_dag import MetaJobsDag
+
+TEAM_NAME: Optional[str] = None
+
+log = logging.getLogger(__name__)
+
+
+def get_json(obj):
+    return json.loads(json.dumps(obj, default=lambda o: o.__dict__))
+
+
+class MetaJobInput(IMetaJobInput):
+    def run_meta_job(self, jobs: List[Dict]):
```

Comment:
it's unusual to have a list of `jobs` being a `List[Dict]`, it should be a list of jobs.

Reply:
That would be handled in https://github.com/vmware/versatile-data-kit/issues/1271